kubernetes-maven-plugin/doc/src/main/asciidoc/inc/_enricher.adoc

```diff
@@ -121,9 +121,14 @@ include::enricher/_jkube_service.adoc[]
 [[jkube-image]]
 ==== jkube-image
 
+include::enricher/_jkube_ingress.adoc[]
+
 [[jkube-name]]
 ==== jkube-name
 
+[[jkube-metadata]]
+include::enricher/_jkube-metadata.adoc[]
```

Comment:
Entry summary should be added to `Table 41. Default Enrichers Overview` 

Entries are sorted alphabetically, this one should go before `jkube-name`